### ISSUE TITLE
ublock lite blocks Joplin Web Clipper

### ISSUE THREAD

**Issue by tbmuc** [uBlockOrigin/uBOL-home#184]
Joplin is a local  running note taking tool with a web clipper extension which needs to access http://127.0.0.1:41190/ping . This is blocked by ublock lite. Adding 127.0.0.1 onto the whitelist does not help. Is there a work around?

**Comment by stephenhawk8054** [comment]
Are you enabling `Block Outsider Intrusion into LAN` list?

**Comment by gorhill** [comment]
Duplicate of <https://github.com/uBlockOrigin/uBOL-home/issues/169>.
